HackerRank/Python/Data Types/Find second largest number in a list.py

```diff
@@ -1,25 +1,13 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-'''
-Problem Statement
+if __name__ == '__main__':
+    n = int(input())
+    arr =list(map(int, input().split(" "))) 
+    k = max(arr)
+    for i in range(0,n):
```

Comment:
@rection  this extra for loop unnecessary, we can directly print arr[1] as this will increase the complexity of the code.

Reply:
yes, you re right.

Reply:
@rection can you keep the existing solution as it is and add this an alternate solution in a new file.